app/backend/src/couchers/urls.py

```diff
@@ -39,6 +39,10 @@ def account_settings_link():
     return f"{config['BASE_URL']}/account-settings"
 
 
+def notification_settings_link():
+    return f"{config['BASE_URL']}/account-settings/notification-settings"
+
+
```

Comment:
The file `app/backend/src/couchers/notifications/background.py`, on line 61, adds a link to "manage notification settings" to the email footer. Could you please update `urls.account_settings_link()` to `urls.notification_settings_link()`!

https://github.com/Couchers-org/couchers/blob/96dc7b746a1f1dacf8e51026ae3993fbb5d42c4a/app/backend/src/couchers/notifications/background.py#L61-L63

Reply:
Done!